### PR TITLE
adding django_debug toolbar

### PR DESCRIPTION
**Pull request by boecklic**
debug_toolbar is a handy add-on that's useful for debugging. I propose to install it since login fails on `bgdi-dev` due to a missing csrf token so it would be useful to debug http headers and stuff

### CODE REVIEW

File: app/middleware/debug_toolbar.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+def check_toolbar_env(request):
+    """ callback to check whether debug toolbar should be shown or not
+
+    for details see
+    https://django-debug-toolbar.readthedocs.io/en/latest/configuration.html#debug-toolbar-config  # pylint: disable=line-too-long
+    """
+    print(os.environ.get('APP_ENV'))
+    if os.environ.get('APP_ENV', 'prod') in ['local', 'dev']:
+        print('blublu')
```
Comment (ltshb) [inline app/middleware/debug_toolbar.py:12]: We should use the logging instead of print statement here with a meaningful message :-)

**Review by ltshb — COMMENTED** [review]
See inline comment

File: app/middleware/debug_toolbar.py (hunk 1 above)
Comment (boecklic) [reply app/middleware/debug_toolbar.py]: :+1: leftover from development, removed it

**Review by ltshb — APPROVED** [review]
:+1:

**Comment by boecklic** [comment]
General question: currently we have none of the dev-tools in the Docker image that's deployed on dev. Does it make sense to deploy a `dev` image on dev env? Cause with the current image the debug_toolbar won't be usable on bgdi-dev

**Review by rebert — APPROVED** [review]

**Comment by boecklic** [comment]
Note: this will go together with this PR https://github.com/geoadmin/infra-kubernetes/pull/23

**Review by ltshb — APPROVED** [review]
:+1:

**Pull request merged**